### ISSUE TITLE
Watched time statistic differs from trakt

### ISSUE THREAD

**Issue by UweTrottmann** [UweTrottmann/SeriesGuide#447]
Reported via email.
[Reported to trakt](https://plus.google.com/+UweTrottmann/posts/idJMdwQaKsF).

A user is seeing a strange difference in release time between SG and trakt:
Trakt.tv: 63 days, 11 hours, 6 minutes watching 2,172 episodes (38 shows)
SeriesGuide: 67 days, 5 hours, 55 minutes watching 2,155 episodes (38 shows)

SG and trakt calculate watched time the same (watched episodes * runtime). Do some testing (mark season watched, check time diffs, etc.).

**Comment by gabrielittner** [comment]
I also see this

Seriesguide: 130 days 12 hours and 55 minutes, 4718 episodes (57 shows)
trakt: 104 days, 6 hours, 15 minutes, 3,936 episodes (57 shows)

If I can help in any way let me know.

**Comment by frabass** [comment]
+1
Seriesguide: 87 days 18 hours and 50 minutes, 3003 episodes (41 shows)
trakt: 79 days, 13 hours, 16 minutes, 2,654 episodes (41 shows)

Tried to flip hide/show special episodes but it gets off even more

This is where stats are computed:
https://github.com/UweTrottmann/SeriesGuide/blob/7124bc6d64cb4c80f51520734432b28e2c4abab5/SeriesGuide/src/main/java/com/battlelancer/seriesguide/ui/StatsFragment.java#L379

Looks to me data comes from:
https://github.com/UweTrottmann/SeriesGuide/blob/7124bc6d64cb4c80f51520734432b28e2c4abab5/SeriesGuide/src/main/java/com/battlelancer/seriesguide/provider/SeriesGuideProvider.java

which in turn loads a SqlLite db, so very likely data in the db are different than data on Trakt.tv db, I'm not sure there's a short fix: SeriesTvGuide allows you to import shows in the app, but you could have multple sources and not have Trakt at all, that said many people including myself only use the Trakt sync feaure to use STG as a client. The fix is to read stats from the Trakt.tv API but that complicates things tremendously cause then you end up with mixed stats, you need to keep track of if a user has that show on Trakt (and potentially do it for other services too).

Bottom line, this is a call Uwe needs to make, if he wants this app to become a Trakt client and provide better support for that or if he wants to retain all the rest of importing modes and so be autonomous.
